integrationTests/IntegrationTest.php

```diff
@@ -272,7 +272,7 @@ public function addFundedCampaignAndCharityToDB(
             isRegularGiving: $isRegularGiving
         );
         ['fundId' => $fundId, 'campaignFundingId' => $campaignFundingId] =
-            $this->addFunding($campaignId, $fundWithAmountInPounds, 1, Pledge::DISCRIMINATOR_VALUE);
+            $this->addFunding($campaignId, $fundWithAmountInPounds, 1, 'pledge');
```

Comment:
Not ideal to be going back to strings in tests like this. But the below changes have an attribute relying on a const `DISCRIMINATOR_MAP`, all outside the class, because of the attribute. I think making that critical code DRYer is a bigger win than avoiding the strings in tests. Open to ideas of how we can cleanly do both.